src/components/yeti/map-layers/IconLayer.vue

```diff
@@ -0,0 +1,201 @@
+      }
+      // then, switch visibility
+      this.onShowLayer();
+    },
+  },
+  created() {
+    this.icon = (color) => {
+      let svg = `<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="-10 -10 20 20">
+          <circle r="9" fill="${color}" />
+          <text y="5" text-anchor="middle" font-family="sans-serif" font-size="14px" font-weight="bold" fill="white">${this.letter}</text>
+        </svg>`;
+      return 'data:image/svg+xml;charset=utf-8,' + encodeURIComponent(svg);
+    };
+
+    this.style = new ol.style.Style({
+      image: new ol.style.Icon({
+        src: this.icon(this.color),
+        size: [20, 20],
+      }),
+    });
+
+    this.overlay = new ol.Overlay({
+      positioning: 'top-left',
+      offset: [-4,-6],
+      className: 'overlay-container',
+    });
+  },
+  mounted() {
+    this.layer = new ol.layer.Vector({
+      source: new ol.source.Vector(),
+      style: this.style,
+    });
+    this.map.addLayer(this.layer);
+
+    this.overlay.setElement(this.$refs.overlay);
+    this.map.addOverlay(this.overlay);
+    this.activeFeature = false;
+
+    // showLayer checked?
+    this.layer.setVisible(this.showLayer);
+
+    // only in first mount
+    if (this.dataLayer.length === 0) {
+      // events
+      Yetix.$on('mapClick', this.onMapClick);
+    }
+  },
+  methods: {
+    onResult(data) {
+      let features = this.getFeaturesFromData(data);
+      let props = this.getPropertiesFromFeatures(features);
+
+      // set data in state
+      this.setState(props);
+
+      // add features
+      this.layer.getSource().addFeatures(features);
+    },
+    onShowLayer() {
+      // set layer visibility
+      this.layer.setVisible(this.showLayer);
+      // if a feature is active (selected) and layer is visible
+      // open overlay, else close
+      if (this.activeFeature && this.showLayer) {
+        this.openOverlay(this.activeFeature);
+      } else {
+        this.closeOverlay();
+      }
+    },
+    onMapClick(e, clickedFeature) {
+      this.closeOverlay(true);
+      if (this.layer.getVisible()) {
+        this.map.forEachFeatureAtPixel(e.pixel, (feature) => {
+          if (feature !== clickedFeature) {
+            return false;
+          }
+          this.$parent.setOverlay(feature);
+          this.activeFeature = feature;
+          this.openOverlay(feature);
+          return true;
+        }, {
+          layerFilter: (layer) => {
+            return layer === this.layer
+          }
+        });
+      }
+    },
+    openOverlay(feature) {
+      let coordinates = feature.getGeometry().getCoordinates();
+      this.overlay.setPosition(coordinates);
+    },
+    closeOverlay(force = false) {
+      this.overlay.setPosition(undefined);
+
+      if (force) {
+        this.$parent.unsetOverlay();
+        this.activeFeature = false;
+      }
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+@import '@/assets/sass/variables.scss';
```

Comment:
You should remove the extension here.

```suggestion
@import '@/assets/sass/variables';
```

Reply:
OK, but why only this specific file? There are many imports with .scss in them...

Reply:
I don't think so?
![image](https://user-images.githubusercontent.com/2234024/197767681-2d09b216-c834-41f2-92a8-1f5d9b27bcdb.png)

It was before, but not since 898c4e43d5f1661161d26e83e16de4b36d9d3a86
Maybe you missed it when rebasing?

Reply:
OK, so I had to remove from all yeti files then! :)

Reply:
Oh, I was checking an old branch, my bad. 
I fix it